src/ChrCopyNumber.cpp

```diff
@@ -658,15 +689,27 @@ void ChrCopyNumber::recalculateLogRatio (float constant) {
 }
 
 void ChrCopyNumber::recalculateRatioWithContam (float contamination, float normGenytype, bool isLogged) { //normGenytype==1 if AB, normGenytype==0.5 if A
-	if (!isLogged) {
-        for (int i = 0; i<length_; i++)
-            if (ratio_[i] != NA) {
-			//ratio_[i] = (ratio_[i]-contamination*normGenytype)/(1-contamination); //correct only for ploidy 2
-                ratio_[i] = (ratio_[i]*(1-contamination+2*contamination/ploidy_) -contamination*normGenytype/ploidy_*2)/(1-contamination);
-
-			if (ratio_[i]<0)
-				ratio_[i] = 0;
+	if (!isLogged) {	
```

Comment:
@ilyasoifer this block changes include mainly prints of ug ratio correction. I think we can keep this for further debugging. it does not affect the run itself.